packages/bpk-docs/src/components/DocsPageBuilder/DocsPageBuilder.js

```diff
@@ -137,6 +130,20 @@ const ComponentExample = (component, registerPlatformSwitchingContent) => {
       })
     : null;
 
+  const androiddocLink = component.androiddocId
```

Comment:
I think these should be `androidDocLink` and `iosDocLink` (uppercase `D`). The sassdoc one is also wrong but I think it's ok to leave that as it would be a lot to change.